addons/spreadsheet/static/src/global_filters/components/filter_value/filter_value.js

```diff
@@ -31,6 +31,7 @@ export class FilterValue extends Component {
         filter: Object,
         model: Object,
         showTitle: { type: Boolean, optional: true },
+        showPlaceholder: { type: Boolean, optional: true },
```

Comment:
Hmm is the placeholder still relevant? Can't we just remove it completely ? It's useless in the side panel as the filter label is repeated twice.
Originally we added the placeholder for dashboard mode. As a consequence it also showed up in the side panel, but it was more of a side effect than a functional need.